### DISCUSSION TITLE
How to delete account

### DISCUSSION THREAD

**Discussion by pegasus2228** [mastodon/mastodon#20617] [Q&A] [answered]
I have an account on baragg.net that I no longer need or want.  I am trying to delete it, but see no option to delete my account.  I have gone to account>settings and the option is not there, only the option to move my data or configure the account.  Am i missing something, or can I have an admin delete my account.  I want to delete the email I am using associated with the account, but am afraid to do so until my baragg account has been deleted.  Any help is appreciated!_

**Comment by ineffyble** [comment, marked as answer]
You'll need to contact the admin for your instance. You can find them here: https://baraag.net/about/more
